src/main/kotlin/ornn/service/InputService.kt

```diff
@@ -0,0 +1,41 @@
+package ornn.service
+
+import ornn.exception.InputIllegalException
+
+object InputService {
+
+    fun getUsersName(): List<String> {
+        return try {
+            val input: String = readLine().toString()
+            validateNullInputException(input)
+            input.split(",")
+        } catch (e: InputIllegalException) {
+            println(e.message)
+            getUsersName()
+        }
+    }
+
+    fun validateNullInputException(input: String) {
+        if (input == "null" || input.contains(" ") || input.isEmpty()) {
+            throw InputIllegalException(Strings.ERROR + Strings.ERROR_NULL_INPUT_EXCEPTION)
+        }
+    }
+
+    fun isUserToTakeMoreCard(name: String): Boolean {
+        println(name + Strings.ASK_TAKE_MORE_CARD)
+        val input = readLine().toString()
```

Comment:
```suggestion
        val input = readLine().toString().lowercase()
```
소문자로 변경해서 아래 검증을 진행하면 확인해야 하는 케이스를 줄일 수 있을 것 같아요! ㅎㅎ